code/lib/types/src/modules/csf.ts

```diff
@@ -135,13 +138,15 @@ export type ViewMode = ViewModeBase | 'story' | 'info' | 'settings' | string | u
 
 type Layout = 'centered' | 'fullscreen' | 'padded' | 'none';
 
-export interface Parameters extends ParametersBase {
-  fileName?: string;
+export interface StorybookParameters {
   options?: Addon_OptionsParameter;
   /** The layout property defines basic styles added to the preview body where the story is rendered. If you pass 'none', no styles are applied. */
   layout?: Layout;
-  docsOnly?: boolean;
-  [key: string]: any;
```

Comment:
Does this mean Parameters are strict by default? That being a possibly big breaking change for users that use custom parameters?

Reply:
I try to make it completely non-breaking, and only add autocompletion for users that do module augmentation. The base Parameter type in CSF is still non-strict:
https://github.com/ComponentDriven/csf/blob/next/src/story.ts#L29-L31